### PR TITLE
Release/automate git tag and release

### PR DESCRIPTION
**Pull request by szuecs**
- refactor auto git tag is done by cdp instead of travis
- feature automatic tagged release page
- feature add binary release with sha256sum.txt and upload to GH release page

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: Makefile [hunk 1]
```diff
@@ -182,35 +182,32 @@ publish-coverage: .coverprofile-all
...
-
-ci-user:
-	git config --global user.email "builds@travis-ci.com"
-	git config --global user.name "Travis CI"
+	make VERSION=$(NEXT_PATCH) tag
 
-ci-release-major: ci-user deps release-major
-ci-release-minor: ci-user deps release-minor
-ci-release-patch: ci-user deps release-patch
+#ci-user:
```
Comment (aryszka) [inline Makefile]: I think we can delete these.

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**